### PR TITLE
Bowie dev dino  or 1   1     

### PR DESCRIPTION
**Pull request by bowie-dev**

### CODE REVIEW

File: bowie_dev_dino__OR_1___1_____.png
Comment (ArjunDandagi) [inline bowie_dev_dino__OR_1___1_____.png:1]: can you remove this file?

Comment (defihook) [reply bowie_dev_dino__OR_1___1_____.png:1]: let me remove

**Review by ni5arga — CHANGES_REQUESTED** [review]
There seems to be two image files, please remove the file which isn't being used.

**Comment by Shrey-Mehra** [comment]
Hey @bowie-dev , As there is no update in the repo from last month I'm closing this Pull request.

**Pull request closed without merging**